build-info-extractor-nuget/src/main/java/org/jfrog/build/extractor/nuget/extractor/NugetRun.java

```diff
@@ -65,23 +79,29 @@
     private String resolutionRepo;
     private String username;
     private String password;
+    private String apiProtocol;
     private String module;
     private String nugetCmdArgs;
     private List<String> dependenciesSources;
     private List<Module> modulesList = new ArrayList<>();
 
     /**
-     * Install npm package.
+     * Run NuGet.
      *
      * @param clientBuilder  - Build Info client builder.
      * @param resolutionRepo - The repository it'll resolve from.
-     * @param nugetCmdArgs   - nuget exec args.
+     * @param useDotnetCli   -
+     * @param nugetCmdArgs   - NuGet exec args.
      * @param logger         - The logger.
-     * @param path           - Path to directory contains package.json or path to '.tgz' file.
+     * @param path           - Path to directory contains .sln file.
```

Comment:
Path to directory contains .sln file. --> Path to the directory containing the .sln file.